UnifiedApi/Client/Services/Submitter/Service.cs

```diff
@@ -578,7 +591,9 @@ private void ResultTask()
   /// <summary>Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.</summary>
   public override void Dispose()
   {
-    CancellationTokenSource.Cancel();
+    Logger.LogWarning($"Wait {CancellationMinutes.Minutes} Minutes for cancellation before shutdown client Service");
```

Comment:
As far as I can see, there is 3 topics in this PR :
- Remove some tests
- Remove the creation of the logger and expecting a logger in input.
- The cancellation of the wait which is not the best way to solve the issue of the results that are not retrieved